cmd/init.go

```diff
@@ -35,22 +33,13 @@ func init() {
 
 func runCmdInit(cmd *cobra.Command, args []string) error {
 	// Validate flags.
-	required := []struct {
-		name, val string
-	}{
-		{"--cluster-name", initOpts.ClusterName},
-		{"--external-dns-name", initOpts.ExternalDNSName},
-		{"--region", initOpts.Region},
-		{"--availability-zone", initOpts.AvailabilityZone},
-	}
-	var missing []string
-	for _, req := range required {
-		if req.val == "" {
-			missing = append(missing, strconv.Quote(req.name))
-		}
-	}
-	if len(missing) != 0 {
-		return fmt.Errorf("Missing required flag(s): %s", strings.Join(missing, ", "))
+	if err := validateRequired(
+		flag{"--cluster-name", initOpts.ClusterName},
+		flag{"--external-dns-name", initOpts.ExternalDNSName},
```

Comment:
Anyone else thinks we should drop `--external-dns-name`,  `--region` and `--availability-zone` and validate later when we know if you are doing HA